src/utils/paymentQRHelper.test.ts

```diff
@@ -0,0 +1,99 @@
+import { Transaction } from "../types";
+import {
+  findValueByKey,
+  updateTransactionsPaymentQRIdentifiers,
+} from "./paymentQRHelper";
+
+describe("tests for paymentQRHelper", () => {
+  describe("tests for findValueByKey", () => {
+    let records: Record<string, unknown>;
+
+    beforeEach(() => {
+      records = {
+        transactionAmount: 10,
+        merchantAccountInformation: {
+          nets: {
+            terminalId: "1234567890",
+            merchantId: "0987654321",
+          },
+        },
+        merchantName: "Merchant name",
+      };
+    });
+
+    it("should find value by key properly", () => {
+      expect.assertions(1);
+      expect(findValueByKey("merchantName", records)).toStrictEqual(
+        "Merchant name"
+      );
+    });
+
+    it("should find nested value by key properly", () => {
+      expect.assertions(1);
+      expect(findValueByKey("terminalId", records)).toStrictEqual("1234567890");
+    });
+
+    it("should return `undefined` if key does not exist", () => {
+      expect.assertions(1);
+      expect(findValueByKey("nonExistentKey", records)).toBeUndefined();
+    });
+  });
+
+  describe("tests for updateTransactionsPaymentQRIdentifiers", () => {
+    let transactions: Array<Transaction>;
+
+    beforeEach(() => {
+      transactions = [
```

Comment:
is this `transactions` before or after the update?

Reply:
This is before the update, will update the property name so it is clearer!